lib/node_modules/@stdlib/math/base/special/bernoullif/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "options": {
+    "task": "build"
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/napi/unary",
+        "stdlib/math/base/assert/is_nonnegative_integer.h",
+        "stdlib/math/base/assert/is_nanf.h",
+        "stdlib/math/base/assert/is_odd.h",
+        "stdlib/constants/float32/ninf.h",
+        "stdlib/constants/float32/pinf.h"
```

Comment:
These paths are all incorrect and are the reason the tests are failing. See other packages and L39 for how to list dependencies.